assets/components/tracking/connector/ConnectorForm.tsx

```diff
@@ -63,173 +63,190 @@ export function ConnectorForm({form, onCreate}: { form: FormInstance, onCreate:
-                        </a>
-                        </Typography.Text>}
-                        rules={[{required: true, message: t`Required`}]}
-                        required
-                    >
-                        <Input autoComplete='off' required placeholder=''/>
-                    </Form.Item>
-
-                    <Form.Item
-                        label={t`Context Value`}
-                        name={['authData', 'context']}
-                        help={<Typography.Text
-                            type='secondary'
-                        >{t`If you not sure, use the default value 4`}
-                        </Typography.Text>}
+                    {
+                        provider === ConnectorProvider.OVHcloud && <>
+                            {
+                                Object.keys(ovhFields).map(fieldName => <Form.Item
+                                    key={ovhFields[fieldName as keyof typeof ovhFields]}
+                                    label={ovhFields[fieldName as keyof typeof ovhFields]}
+                                    name={['authData', fieldName]}
+                                    rules={[{required: true, message: t`Required`}]}
+                                >
+                                    <Input autoComplete='off'/>
+                                </Form.Item>)
+                            }
+                            <Form.Item
+                                label={t`OVH Endpoint`}
+                                name={['authData', 'apiEndpoint']}
+                                rules={[{required: true, message: t`Required`}]}
+                            >
+                                <Select options={ovhEndpointList} optionFilterProp='label'/>
+                            </Form.Item>
+                            <Form.Item
+                                label={t`OVH subsidiary`}
+                                name={['authData', 'ovhSubsidiary']}
+                                rules={[{required: true, message: t`Required`}]}
+                            >
+                                <Select options={ovhSubsidiaryList} optionFilterProp='label'/>
+                            </Form.Item>
 
-                        required={false}
-                    >
-                        <Input autoComplete='off' required={false} placeholder='4'/>
-                    </Form.Item>
+                            <Form.Item
+                                label={t`OVH pricing mode`}
+                                name={['authData', 'pricingMode']}
+                                rules={[{required: true, message: t`Required`}]}
+                            >
+                                <Popconfirm
+                                    title={t`Confirm pricing mode`}
+                                    description={t`Are you sure about this setting? This may result in additional charges from the API Provider`}
```

Comment:
```suggestion
                                    description={t`Are you sure about this setting? This may result in additional charges from OVHCloud`}
```